packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -64,6 +65,7 @@ export function makeRumPublicApi(
 
   let getInternalContextStrategy: StartRumResult['getInternalContext'] = () => undefined
   let getInitConfigurationStrategy = (): InitConfiguration | undefined => undefined
+  let stopSessionStrategy: () => void = noop
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** we didn't discuss what should happen when calling `stopSession()` before RUM is started. We might want to do something before GA